spyderlib/utils/icon_manager.py

```diff
@@ -40,7 +40,7 @@
     'arrow-continue':          [('spyder.continue',), {'color': '#3775a9'}],
     'arrow-step-in':           [('spyder.step-into',), {'color': '#3775a9'}],
     'arrow-step-out':          [('spyder.step-out',), {'color': '#3775a9'}],
-    'stop_debug':              [('spyder.stop',), {'color': '#3775a9'}],
+    'stop_debug':              [('fa.stop',), {'color': '#3775a9'}],
```

Comment:
The Spyder 2 theme is the same as before.

The Spyder 3 theme is a mix of font-awesome and the custom Spyder font which has some icons wh8ch are not in font-awesome.